website/src/content/development/forms.mdx

```diff
@@ -610,43 +610,51 @@ Sometimes a form contains many different fields, resulting in the form component
 
 One way to improve that is to split the form component into multiple smaller components. As a rule of thumb you can create one separate component for each form field. For example a `<FormChannelNameField>`, a `<FormChannelKeyField>`, and so on.
 
-These components are simple wrapper components of the UI Kit field components but they contain and encapsulate all the logic related to the specific field. As a result, your form component can look like this:
+<Info>
+
+To use the `useField` hook the form must be initialized using the `<Formik>` component instead of using the `useFormik` hook, so that the React context is properly defined.
+
+</Info>
+
+As a result, the form component can look like this:
 
 ```jsx title="channels-form.js"
-import { useFormik } from 'formik';
-import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import { Formik } from 'formik';
 import Spacings from '@commercetools-uikit/spacings';
 import FormChannelNameField from './form-channel-name-field';
 import FormChannelKeyField from './form-channel-key-field';
 import validate from './validate';
 
 const ChannelsForm = (props) => {
-  const dataLocale = useApplicationContext((context) => context.dataLocale);
-  const formik = useFormik({
-    // Pass initial values from the parent component.
-    initialValues: props.initialValues,
-    // Handle form submission in the parent component.
-    onSubmit: props.onSubmit
-    validate,
-    enableReinitialize: true,
-  });
-
-  // Only contains the form elements, no buttons.
-  const formElements = (
-    <Spacings.Stack scale="l">
-      <FormChannelNameField />
-      <FormChannelKeyField />
-    </Spacings.Stack>
+  return (
+    <Formik
```

Comment:
I would suggest to also wrap the field with a `form` element as suggested in [Formik's docs](https://formik.org/docs/api/formik) as it is more semantic.
Also, perhaps we could add a submit button for a better complete form example.

Reply:
I totally agree, but it seems to me the whole page deserves revisiting and adding `form` and the submit button wherever needed. There is a `// Only contains the form elements, no buttons.` comment in multiple places and either it should be addressed everywhere or left as it is. But it seems it's out of the scope of this PR. I only wanted to fix the issue with Formik context not being present when we use `useField`

Reply:
Yes, I also agree then that it is out of this PR's scope.
Perhaps something to look into as a follow-up.